CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## [1.1.0]
+
+- Fix: Color Icon buttons are bigger to improve accessibility
```

Comment:
You should add your changes here, not the same as in 1.0.2.

Reply:
Sorry, wanted to update it but forgot - will do